<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlTableWin</class>
 <widget class="QWidget" name="SqlTableWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="6">
      <widget class="QPushButton" name="pbnSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QPushButton" name="pbnRevert">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="7">
      <widget class="QTableView" name="sqlTableView"/>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="bnDelete">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="bnAdd">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leFilter"/>
       </item>
       <item>
        <widget class="QToolButton" name="bnSetFilter">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="bnUnSetFilter">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slotAddRow()</slot>
  <slot>slotDelRow()</slot>
 </slots>
</ui>
